tap_core/src/rav.rs

```diff
@@ -1,12 +1,39 @@
 // Copyright 2023-, Semiotic AI, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-//! Module containing Receipt type used for providing and verifying a payment
+//! # Receipt Aggregate Voucher
 //!
-//! Receipts are used as single transaction promise of payment. A payment sender
-//! creates a receipt and ECDSA signs it, then sends it to a payment receiver.
-//! The payment receiver would verify the received receipt and store it to be
-//! accumulated with other received receipts in the future.
+//! Receipts Aggregate Voucher or RAV is the struct that is sent to the
+//! blockchain to redeem the aggregated receipts. Receipts are aggregated
+//! into a single RAV via a [`RAVRequest`] and then sent to `tap_aggregator`.
+//! The request is verified and signed by the aggregator and the response
+//! is stored on the indexer side.
+//!
+//! Every time you have enough receipts to aggregate, you can send another
+//! RAV request to the aggregator. The aggregator will verify the request and
+//! increment the total amount of GRT that has been aggregated.
```

Comment:
tap_core is not GRT-specific
```suggestion
//! increment the total amount that has been aggregated.
```